tests/unit/test_fomo.py

```
@@ -0,0 +1,800 @@
+        from libreyolo.utils.serialization import wrap_libreyolo_checkpoint
+        ckpt_dict = wrap_libreyolo_checkpoint(
+            model.model.state_dict(),
+            model_family="fomo",
+            size="s",
+            task="point",
+            nc=1,
+        )
+        torch.save(ckpt_dict, dummy_ckpt)
+        
+        class DummyTrainer:
+            def __init__(self, *args, **kwargs):
```

Ruff (0.15.17)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)